src/buffer_object.rs

```diff
@@ -448,15 +465,8 @@ pub struct WrongDeviceError;
 
 impl fmt::Display for WrongDeviceError {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
-        use std::error::Error;
-        write!(f, "{}", self.description())
+        "The gbm specified is not the one this buffer object belongs to".fmt(f)
```

Comment:
`f.write_str("the string")` might be nicer, though that same function should be used under the hood when calling `Display::fmt` on the string.